newdemojail.yaml

```diff
@@ -0,0 +1,772 @@
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "DemoJailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "BookId"
+      incident.startdatetime:
+        type: "incident.startdatetime"
+        column: "SentenceStartDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ['yyyy-MM-dd']
+          timezone: "America/Los Angeles"
+      ol.projectedreleasedate:
+        type: "ol.projectedreleasedatetime"
+        column: "SentenceEndDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ['yyyy-MM-dd']
+          timezone: "America/Los Angeles"
+      ol.releasedate:
+        type: "ol.releasedatetime"
+        column: "ReleaseDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ['yyyy-MM-dd']
+          timezone: "America/Los Angeles"
+      criminaljustice.releasetype:
+        type: "criminaljustice.releasetype"
+        column: "ReleaseType"
+    name: "jailstay"
+  jailstaylength:
+    fqn: "ol.jailstaylength"
+    entitySetName: "DemoJailstaylength"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "JailstayLength"
+      criminaljustice.timeserveddays:
+        type: "criminaljustice.timeserveddays"
+        column: "JailstayLength"
+        transforms:
+        - !<transforms.ParseIntTransform> {}
+    name: "jailstaylength"    
+  courtcase:
+    fqn: "justice.case"
+    entitySetName: "DemoCourtCase"
+    propertyDefinitions:
+      j.CaseNumberText:
+        type: "j.CaseNumberText"
+        column: "DocketNo"
+      justice.courtcasenumber:
+        type: "justice.courtcasenumber"
+        column: "DocketNo"
+    name: "courtcase"
+  hearing:
+    fqn: "justice.case"
+    entitySetName: "DemoHearing"
+    propertyDefinitions:
+      j.CaseNumberText:
+        type: "j.CaseNumberText"
+        column: "DocketNo"
+      justice.courtcasenumber:
+        type: "justice.courtcasenumber"
+        column: "DocketNo"
+      general.datetime:
+        type: "general.datetime"
+        column: "HearingDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ['yyyy-MM-dd']
+          timezone: "America/Los Angeles"
+      ol.type:
+        type: "ol.type"
+        column: "HearingType"
+    name: "hearing"
+  mentalhealthstatus:
+    fqn: "ol.healthcondition"
+    entitySetName: "DemoMentalHealth"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "MentalHealthStatus"
+      ol.description:
+        type: "ol.description"
+        column: "MentalHealthStatus"
+    name: "mentalhealthstatus"
+  sexoffenderstatus:
+    fqn: "ol.sexoffender"
+    entitySetName: "DemoSexoffenderstatus"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SexOffender"
+      ol.registeredflag:
+        type: "ol.registeredflag"
+        column: "SexOffender"
```

Comment:
needs boolean transform?

Reply:
Need to review this with Joke - we are trying to completely de-transform so I will just convert to 1/0 if this is the case